tests/foreman/cli/test_contentaccess.py

```diff
@@ -76,25 +76,22 @@ def rh_repo_setup_ak(module_sca_manifest_org, module_target_sat):
 def vm(
     rh_repo_setup_ak,
     module_sca_manifest_org,
-    rhel7_contenthost_module,
+    rhel9_contenthost_module,
```

Comment:
Could we use `module_rhel_contenthost` and parametize the test with `@pytest.mark.rhel_ver_match('9')` instead of adding new `rhelX_contenthost` fixture? They are going to be removed.

```suggestion
    module_rhel_contenthost,
```